pkg/runners/builtin_runner/range_check.go

```diff
@@ -0,0 +1,72 @@
+package builtinrunner
+
+import (
+	"errors"
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+var ErrRangeOutOfBounds = errors.New("RangeCheckNumOutOfBounds")
+
+const CHECK_RANGE_BUILTIN_NAME = "range_check"
+const INNER_RC_BOUND_SHIFT = 16
+const INNER_RC_BOUND_MASK = math.MaxUint16
+const CELLS_PER_RANGE_CHECK = 1
+
+const N_PARTS = 8
+
+type RangeCheckBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func NewRangeCheckBuiltinRunner(included bool) *RangeCheckBuiltinRunner {
+	return &RangeCheckBuiltinRunner{
+		base:     memory.NewRelocatable(0, 0),
```

Comment:
We can omit this value as it will take the zero value, which is (0,0) for a Relocatable

Reply:
But when we add segments to the builtin, it takes the value of the new segment added, why the should not have it as attribute?

Reply:
Yes, it should be an attribute, but we don't need to create it explicitly when creating a `RangeCheckBuitinRunner`, we can omit the field and it will take the zero value when creating the struct